moredis/dbs.go

```diff
@@ -30,6 +39,34 @@ func SetupDbs(mongoURL, redisURL string) (*mgo.Database, redis.Conn, error) {
 	return mongoDB, redisConn, nil
 }
 
+// resolveRedis takes in a redis address and checks for a sentinel:// prefix to resolve. If one is present, it uses
+// sentinel to resolve the master and returns that. Otherwise, it returns the original address.
+func resolveRedis(address string) (string, error) {
+	if !strings.HasPrefix(address, "sentinel://") {
+		return address, nil
+	}
+
+	match := regexp.MustCompile(`sentinel://([^/]+)/(.*)`)
+	addressParts := match.FindAllStringSubmatch(address, -1)
+
+	if len(addressParts) < 1 || len(addressParts[0]) < 3 {
```

Comment:
It's a slice of 'matches' where a match is a slice of strings where the string at index 0 is the entire match and every subsequent string is the next group in the match.

For example, the regex: `i(.)` against the string `trolling is a art.`, would return:
 `[ ['in', 'n'], ['is', 's'] ]`